### PR TITLE
[WIP] Fix order dependence in specs.

### PR DESCRIPTION
**Pull request by tpendragon**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,3 +11,6 @@ group :development, :test do
   gem 'pry' unless ENV['CI']
   gem 'pry-byebug' unless ENV['CI']
 end
+
+gem 'active-fedora', github: 'projecthydra/active_fedora', branch: 'master'
```
Comment (cbeer) [inline Gemfile]: Merge now, or wait until we can pin these in the gemspec?

Comment (tpendragon) [inline Gemfile]: I'd wait until we can pin. Just getting the stack all sorted.

Comment (cbeer) [inline Gemfile]: 👍 good answer.

**Comment by cjcolvar** [comment]
This looks like a good PR.  Is the active-fedora and hydra-works pinning resolvable now?

**Comment by tpendragon** [comment]
This was fixed at some point in the last couple years, so isn't relevant anymore.

**Pull request closed without merging**